module/userProjects/helper.js

```
@@ -747,8 +747,10 @@ module.exports = class UserProjectsHelper {
 					solution = await solutionsQueries.createSolution(solution)
 					solutionsCreated.push(solution)
 				}
-				const solutionIds = solutionsCreated.map((solution) => {
-					return solution._id
+
+				let componentLength = userPrivateProgram.components.length
+				const componentArray = solutionsCreated.map((solution) => {
+					return { _id: new ObjectId(solution._id), order: ++componentLength }
```

ESLint

[error] 'ObjectId' is not defined.

(no-undef)